gst/gst_bus.go

```diff
@@ -300,7 +308,7 @@ func (b *Bus) SetSyncHandler(f BusSyncHandler) {
 		b.Instance(),
 		C.GstBusSyncHandler(C.cgoBusSyncHandler),
 		(C.gpointer)(unsafe.Pointer(ptr)),
-		nil,
+		C.GDestroyNotify(C.cgoDestroyBusWatch),
```

Comment:
You probably want to check if there's any other function that takes a `GDestroyNotify` and you're currently passing `nil` there. Almost certainly that has the same bug.

Reply:
Yeah that's a good idea, although a lot of work I think. I took a note here: #112